pkg/proto/agentbaker/v1/config.proto

```diff
@@ -8,56 +8,58 @@ message Configuration {
   
   string cluster_certificate_authority = 1;
 
-  // only required until Secure TLS bootstrapping in place
-  // would use kubelet identity after that.
-  string tls_bootstrap_token = 2;
-
   // cluster/user config
-  string kubernetes_version = 3; // Q: can this be auto-detected? Or is this part of specifying the desired node version?
+  string kubernetes_version = 2; // Q: can this be auto-detected? Or is this part of specifying the desired node version?
     // removed kubernetesHyperkubeSpec
-  string kube_binary_url = 4;      // maybe we can combine this and custom_kube_binary_url later once we figure out the logic of how they are used.
-  string custom_kube_binary_url = 5;
-  string kubeproxy_url = 6;
-  ApiServerConfig apiserver_config = 7;
-  string subscription_id = 8;
-  string resource_group = 9;
-  string location = 10;
-  string vm_type = 11;  
+  string kube_binary_url = 3;      // maybe we can combine this and custom_kube_binary_url later once we figure out the logic of how they are used.
+  string custom_kube_binary_url = 4;
+  string kubeproxy_url = 5;
+  ApiServerConfig apiserver_config = 6;
+  string subscription_id = 7;
+  string resource_group = 8;
+  string location = 9;
+  string vm_type = 10;  
   
-  string primary_availability_set = 12;
-  string primary_scale_set = 13;
+  string primary_availability_set = 11;
+  string primary_scale_set = 12;
   
-  CloudProviderConfig cloud_provider_config = 14;
-  IdentityConfiguration identity_configuration = 15;
+  CloudProviderConfig cloud_provider_config = 13;
+  IdentityConfiguration identity_configuration = 14;
 
-  RuncConfig runc_config = 16;
+  RuncConfig runc_config = 15;
 
   // ... and a bunch of user-specified config
-  string repo_depot_endpoint = 17;
-  string linux_admin_username = 18;
+  string repo_depot_endpoint = 16;
```

Comment:
since this doesn't need to be configured (only based on SKU), we could consider removing this too